packages/build-info/package.json

```diff
@@ -57,7 +57,7 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.30.0",
-    "@types/node": "^14.18.53",
+    "@types/node": "^18.14.0",
```

Comment:
Should this be pinned? With `^`, `@types/node@18.19.103` seems to be installed, which might have references to features 18.14 doesn't have? (not sure if there are any like that, but feels like this should be locked down to min version we declare to support)

Reply:
It seems to indeed be the case that the minor/patch versions are a best effort match, and not a guarantee. They mention on their docs that due to being a community run project, there are times when the type implementations can lag behind by several weeks, or be incorrect for a time until patched.

Although ideally there is a 1:1 match with the version of node we're using, seeing as that's not necessarily the case, I'm now leaning towards the original form of this PR, and just opting for the latest `18.*` version of the type library.